plugins/tech-radar/src/components/RadarEntry/RadarEntry.tsx

```diff
@@ -82,9 +100,32 @@ const RadarEntry = (props: Props): JSX.Element => {
       onClick={onClick}
       data-testid="radar-entry"
     >
-      <WithLink url={url} className={classes.link}>
-        {blip}
-      </WithLink>
+      {' '}
+      {description ? (
+        <>
+          <a
+            className={classes.link}
+            onClick={handleClickOpen}
+            href={url ? url : '#'}
+            role="button"
+            tabIndex={0}
+            onKeyPress={toggle}
+          >
+            {blip}
+          </a>
+          <RadarDescription
```

Comment:
Hm I'm concerned that we'd be rendering hundreds of (closed) dialogs, feels like the parent should hold a single dialog that gets toggled on click with some certain data in it instead :)

Like, having a useState that holds undefined or a radar entry, and then the dialog `open` prop is connected to whether the state is undefined or not...that kind of thing

Reply:
Thanks for the review, I think it only renders to the dom when actually in the open state from what I can tell but I catch your drift and can try to rework it.  I'm not really sure how to move the dialog into the parent, if there is some pattern that is usually followed I could give it a go.

Reply:
I figured since I don't need the component unless open=true that I would just not render it unless open=true